src/main/java/com/softserve/sportshub/utils/JwtUtils.java

```diff
@@ -1,8 +1,25 @@
 package com.softserve.sportshub.utils;
 
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
 import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+import javax.servlet.http.HttpServletRequest;
 
 public class JwtUtils {
     //TODO: global file: JWT secret
     public static Algorithm algorithm = Algorithm.HMAC256("secret".getBytes());
+
+    public static String getUsernameFromTokenInsideRequest (HttpServletRequest request) throws Exception {
+        String authHeader = request.getHeader("Authorization");
+        if (authHeader != null && authHeader.startsWith("Bearer ")) {
+            String refresh_token = authHeader.substring("Bearer ".length());
```

Comment:
right, I just did it on purpose as headers in chrome tab looks like this

cache-control: max-age=0, private, must-revalidate
content-encoding: gzip
etag: W/"a4da762ed5c0fc2fd90fba88797856a0"
expect-ct: max-age=2592000, report-uri="https://api.github.com/_private/browser/errors"
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
server: GitHub.com

<img width="294" alt="Zrzut ekranu 2022-10-19 o 10 20 28" src="https://user-images.githubusercontent.com/45168539/196637019-21e080e0-e70b-4255-967a-04995c77120b.png">


But you are right, it is Java code so there should be camelCase instead of snake_case